packages/core/src/Breadcrumb.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Icon from './Icon'
+import Text from './Text'
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Breadcrumb({ className, isCurrent, isCompleted, children, onClick }) {
+  const color = isCurrent || isCompleted ? 'primary' : 'text.light'
+
+  return (
+    <Container alignItems="center" onClick={onClick}>
+      {isCompleted && <CheckIcon />}
+      <Text color={color} bold={isCurrent} fontSize={1}>
+        {children}
+      </Text>
+    </Container>
+  )
+}
+
+const CheckIcon = () => (
+  <Icon name="Check" color="primary" size={16} mr={1} legacy={false} />
+)
+
+Breadcrumb.displayName = 'Breadcrumb'
+
+Breadcrumb.propTypes = {
+  className: PropTypes.string,
+  isCurrent: PropTypes.bool,
+  isCompleted: PropTypes.bool,
+  children: PropTypes.node,
+  onClick: PropTypes.func
+}
+
+Breadcrumb.defaultProps = {
+  className: ''
+}
```

Comment:
I think you can omit the default prop for `className`. Also, don't forget to pass `className` down to `Container`.